src/core/nis1/KeyPair.ts

```diff
@@ -1 +1,96 @@
-export class KeyPair {}
+/*
+ * Copyright 2021 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as Crypto from 'crypto';
+import { Converter } from '../utils/Converter';
+import { keccakHash, KeccakHasher } from '../utils/Utilities';
+const Ed25519 = require('./external/nacl-fast.js').lowlevel;
+
+export class KeyPair {
```

Comment:
Check the current Keypair abstract,  the nis1 keyPair class should extend the global one (similar to the SymbolKeyPair) ??